kitchen/wazuh-puppet/Dockerfile

```diff
@@ -24,6 +26,14 @@ RUN apt-get update -qq -y && \
       wget \
       zlib1g-dev
 
+
+# PDK dependencies
+RUN wget https://apt.puppet.com/puppet-tools-release-bionic.deb && \
+    dpkg -i puppet-tools-release-bionic.deb && \
+    apt-get update && \
+    apt-get install pdk
```

Comment:
This should be `apt-get install -y pdk` to ensure no human interaction is required.

Reply:
Hi @xr09 !

Change has been applied and tested.

Kr,

Rshad